app/views/collections/_filters.html.erb

```diff
@@ -1,70 +1,54 @@
+<%= form_for @search, as: :collection_search, url: collections_path, html: { method: :get, class: "narrow-hidden filters", id: "collection-filters" } do |f| %>
   <h3 class="landmark heading"><%= ts("Filters") %></h3>
   <%= field_set_tag ts("Filter collections:") do %>
     <dl>
       <dt class="landmark"><%= ts("Sort and Filter") %></dt>
       <dd class="submit actions"><%= submit_tag ts("Sort and Filter") %></dd>
       <dt class="sort">
-        <%= label_tag :sort_column, ts("Sort by") %>
+        <%= f.label :sort_column, ts("Sort by") %>
       </dt>
       <dd class="sort">
-        <%= select_tag :sort_column,
-            options_for_select({
-              ts("Title") => "collections.title",
-              ts("Date Created") => "collections.created_at" },
-              params[:sort_column]) %>
+        <%= f.select :sort_column, options_for_select({ ts("Title") => "title.keyword", ts("Date Created") => "created_at" }, "created_at") %>
       </dd>
       <dt class="sort">
-        <%= label_tag :sort_direction, ts("Sort direction") %>
+        <%= f.label :sort_direction, ts("Sort direction") %>
       </dt>
       <dd class="sort">
-        <%= select_tag :sort_direction,
-            options_for_select({
-              ts("Ascending") => "ASC",
-              ts("Descending") => "DESC" },
-              params[:sort_direction]) %>
+        <%= f.select :sort_direction, options_for_select({ ts("Ascending") => "ASC", ts("Descending") => "DESC" }, "DESC") %>
       </dd>
 
       <dt class="autocomplete search">
-        <%= label_tag "collection_filters_title",
-            ts("Filter by title or name") %>
+        <%= f.label "title", ts("Filter by title") %>
       </dt>
       <dd class="autocomplete search">
-        <%= text_field_tag "collection_filters[title]",
-            params[:collection_filters][:title],
-            autocomplete_options("collection_fullname",
-              data: { autocomplete_token_limit: 1 }) %>
+        <%= f.text_field "title", autocomplete_options("collection_fullname", data: { autocomplete_token_limit: 1 }) %>
       </dd>
+
       <dt class="autocomplete search">
-        <%= label_tag "collection_filters_fandom", ts("Filter by fandom") %>
+        <%= f.label "tag", ts("Filter by tag") %>
       </dt>
       <dd class="autocomplete search">
-        <%= text_field_tag "collection_filters[fandom]",
-            params[:collection_filters][:fandom],
-            autocomplete_options("fandom", data: { autocomplete_token_limit: 1 }) %>
+        <%= f.text_field "tag", autocomplete_options("tag", data: { autocomplete_token_limit: 5 }) %>
       </dd>
+
       <dt><%= ts("Closed") %></dt>
       <dd>
         <ul>
           <li>
-            <%= label_tag "collection_filters_closed_true" do %>
-              <%= radio_button_tag "collection_filters[closed]", true,
-                  params[:collection_filters][:closed] == "true" %>
+            <%= f.label "closed_true" do %>
```

Comment:
I think we should do
```suggestion
            <%= f.label "closed_true", value: true do %>
```
for the labels for the radio buttons. Refer to https://api.rubyonrails.org/classes/ActionView/Helpers/FormBuilder.html#method-i-label and how it's used in the works filters